src/DiceRoller/Dice.cpp

```diff
@@ -0,0 +1,121 @@
+//
+//  Dice.cpp
+//  DiceRoller
+//
+//  Created by Francesco Benzi on 2019-09-20.
+//  Copyright © 2019 Francesco Benzi. All rights reserved.
+//
+
+#include "Dice.hpp"
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+Dice::Dice(){
+    
+    totalRolls = 0;
+    double rollPercentageArray[6];
```

Comment:
These will have to be initialized with "new" with the pointers/

Reply:
I updated these according to the format you used in your .cpp files